test/NuGet.Clients.Tests/NuGet.CommandLine.Test/CommandRunner.cs

```diff
@@ -29,6 +26,7 @@ public class CommandRunner
                 RedirectStandardError = true,
                 RedirectStandardInput = inputAction != null
             };
+            psi.EnvironmentVariables["NuGetTestModeEnabled"] = "True";
```

Comment:
In the PowerShell script: https://github.com/NuGet/NuGet.Client/pull/135/files#diff-3abd5921200b239022d9c6e0240a9692R35